packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -184,13 +184,43 @@ interface EncodedFieldChange {
 }
 
 export class ModularEditBuilder extends ProgressiveEditBuilder<FieldChangeMap> {
+    private readonly fieldKinds: ReadonlyMap<FieldKindIdentifier, FieldKind>;
+
     constructor(
         family: ModularChangeFamily,
         deltaReceiver: (delta: Delta.Root) => void,
         anchors: AnchorSet,
     ) {
         super(family, deltaReceiver, anchors);
+        this.fieldKinds = family.fieldKinds;
     }
 
-    // TODO: Finish implementation
+    submitChange(
```

Comment:
Consider adding TSDoc for the parameters.

Reply:
Done

Reply:
Eslint is sad about the lack of hyphens :)